src/m365/spo/commands/homesite/homesite-set.spec.ts

```diff
@@ -101,31 +86,51 @@ describe(commands.HOMESITE_SET, () => {
         siteUrl: "https://contoso.sharepoint.com/sites/Work"
       }
     } as any);
-    assert(loggerLogSpy.calledWith('The Home site has been set to https://contoso.sharepoint.com/sites/Work.'));
+    assert(loggerLogSpy.calledWith(expected));
```

Comment:
While this is the way we used to write tests, it doesn't really test very much. Right now we are only testing whether we are logging the response that we get  from the API.
A better test would be to actually check which request body was sent to the API.
You can find an example here: https://github.com/pnp/cli-microsoft365/blob/f8f3553c972f6064df03ea17e193be07b2b77d79/src/m365/aad/commands/user/user-guest-add.spec.ts#L141-L153

I suggest that we refactor your 2 tests to check the request body. And add 1 extra test that checks whether the API response is correctly logged (like you do now).